### PR TITLE
new option to insert the number of the line

### PR DESCRIPTION
**Pull request by guy-maurel**
at the beginning of each line
for developer only

### CODE REVIEW

**Review by micheleCTDEAdmin — CHANGES_REQUESTED** [review]
New options needs to be added to default.cfg and related files too.

File: documentation/htdocs/config.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Uncrustify-0.75.0
+# Uncrustify_d-0.75.0-60-a7a9d652-dirty
```
Comment (micheleCTDEAdmin) [inline documentation/htdocs/config.txt:1]: Probably this change was unintentional...

File: documentation/htdocs/default.cfg [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Uncrustify-0.75.0
+# Uncrustify_d-0.75.0-60-a7a9d652-dirty
```
Comment (micheleCTDEAdmin) [inline documentation/htdocs/default.cfg:1]: Probably this change was unintentional...

File: etc/defaults.cfg [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Uncrustify-0.75.0
+# Uncrustify_d-0.75.0-60-a7a9d652-dirty
```
Comment (micheleCTDEAdmin) [inline etc/defaults.cfg:1]: Probably this change was unintentional...

File: etc/uigui_uncrustify.ini [hunk 4]
```diff
@@ -15,20 +15,20 @@ parameterOrder=ipo
 showHelpParameter=-h
 stringparaminquotes=false
 useCfgFileParameter="-c "
-version=Uncrustify-0.75.0
+version=Uncrustify_d-0.75.0-60-a7a9d652-dirty
```
Comment (micheleCTDEAdmin) [inline etc/uigui_uncrustify.ini:18]: Probably this change was unintentional...

**Review by micheleCTDEAdmin — CHANGES_REQUESTED** [review]
Minor touch up required related to version number in cfg file.

**Comment by micheleCTDEAdmin** [comment]
> reflect different things, exactly what we have (a new version)

Yes, I figured you used the /release_tool.py script.
I agree the current version is newer that 0.75, but I don't think it is good to add a hash reference in the config files, especially a temporary one like -dirty. Maybe we can use something like '# Uncrustify_d-0.76.0-development' to indicate that it is a version that will end up in 0.76_f but it is still in development? What do you think?

**Comment by guy-maurel** [comment]
Yes, I used release_tool.py.
But, using uncrustify  --show-config produce the same line.
Your proposal is fine.
Could you integrate it in the script?
I'll make the change per editor, for just now.

**Review by micheleCTDEAdmin — APPROVED** [review]
Looks good.

**Comment by micheleCTDEAdmin** [comment]
> Could you integrate it in the script?

I have added this in my TODO list. Will do at some point in coming weeks.

**Pull request merged**